### DISCUSSION TITLE
Integración del Webhook de WhatsApp en Node.js

### DISCUSSION THREAD

**Discussion by connuna** [community/community#111417] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

¿Alguien aquí se ha aventado a integrar WhatsApp con Node.js? ¿Qué rollo han tenido con eso? Me encantaría oír sus experiencias, ¿qué tipo de aplicaciones han armado y cómo han aprovechado el poder de WhatsApp para mejorar la comunicación?

**Comment by madmarin** [comment, marked as answer]
Usa Twilio para que puedas integrar whatsapp con node, un tiempo lo use y va de maravilla pero recuerda que cobra comiciones 

`const express = require('express');
const bodyParser = require('body-parser');

const app = express();
const port = 3000;

app.use(bodyParser.urlencoded({ extended: false }));

app.post('/webhook', (req, res) => {
  const message = req.body.Body;
  const sender = req.body.From;
  
  console.log(`Mensaje recibido de ${sender}: ${message}`);
  
  // Aquí puedes agregar la lógica para procesar el mensaje recibido
  
  res.sendStatus(200);
});

app.listen(port, () => {
  console.log(`Servidor corriendo en http://localhost:${port}`);
});
`

**Reply by connuna** [reply]
Muchas gracias

**Comment by Unzurna04** [comment]
Nice

**Comment by rizwansan4-spec** [comment]
![Uploading Screenshot_20260424_000748_YoHo.png…]()

**Comment by rizwansan4-spec** [comment]
June ke do

**Comment by rizwansan4-spec** [comment]
8jchfndjvcfu6

**Comment by ahmedtrechri-oss** [comment]
Hi
